### PR TITLE
Styleupdates

### PR DESCRIPTION
**Pull request by josh-lew**
display stars function added and unit tested to change book and destination rating into a series of images on index.php page. CSS updates for readability and responsivity

### CODE REVIEW

File: addBook.php [hunk 1]
```diff
@@ -31,14 +31,16 @@
   } else {
       echo 'Please add all the specified guidebook details';
   }
-
 }
+
+
```
Comment (erc-clarkson) [inline addBook.php:36]: remove empty spaces

File: normalize.css [hunk 2]
```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
+   ========================================================================== */
+
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in iOS.
```
Comment (erc-clarkson) [inline normalize.css:8]: remove this comment

File: style.css [hunk 3]
```diff
@@ -17,32 +18,138 @@ nav {
...
-
 }
 
 .bookContainer {
     text-align: center;
     padding-top: 50px;
     background-color: #9bc1bc;
+    border: 5px solid black ;
+    padding-bottom: 10px;
+
```
Comment (erc-clarkson) [inline style.css:77]: extra line

File: style.css [hunk 4]
```diff
@@ -17,32 +18,138 @@ nav {
...
+@media screen and (max-width: 1300px) {
+
+.bookGrid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 10px;
+    margin-top: 10px;
+    margin-left: 10px;
 }
```
Comment (erc-clarkson) [inline style.css:106]: add indentation to bracket

**Review by erc-clarkson — COMMENTED** [review]
looking good!

**Review by erc-clarkson — APPROVED** [review]

File: addBook.php [hunk 5]
```diff
@@ -31,14 +31,16 @@
   } else {
       echo 'Please add all the specified guidebook details';
   }
-
 }
+
```
Comment (mporam) [inline addBook.php:35]: why?

**Review by mporam — CHANGES_REQUESTED** [review]

**Pull request merged**